Source/WebCore/svg/SVGSVGElement.cpp

```diff
@@ -432,8 +432,10 @@ RenderPtr<RenderElement> SVGSVGElement::createElementRenderer(RenderStyle&& styl
 {
     if (isOutermostSVGSVGElement()) {
 #if ENABLE(LAYER_BASED_SVG_ENGINE)
-        if (document().settings().layerBasedSVGEngineEnabled())
+        if (document().settings().layerBasedSVGEngineEnabled()) {
+            document().setMayHaveRenderedSVGRootElements();
```

Comment:
Why may have rendered? Don't we know at this point at least one is created?

Reply:
Yeah, and eventually it is removed again from DOM -- this we don't track per document...  It is sufficient as heuristic though, to check if we want to dump with sub pixel precision or not.

Reply:
Still sounds vague to me. setHasEverRenderedSVGRootElements?

Reply:
Well then it would deviate from "setMayHaveRenderedForeignObjects" - which is recently added logic. I wanted to stay with that convention.

Reply:
Hmm, I see, I guess I missed that at the time. Anyway this is not a big deal.